src/main/java/frc/robot/subsystems/drive/Drive.java

```diff
@@ -301,4 +304,18 @@ public double getMaxLinearSpeedMetersPerSec() {
   public double getMaxAngularSpeedRadPerSec() {
     return maxSpeedMetersPerSec / driveBaseRadius;
   }
+
+  public void checkForImpact() {
+    /*double mDifference = Math.abs(gyroInputs.VectorM - gyroInputs.pVectorM);
+    double aDifference = Math.abs(gyroInputs.VectorA - gyroInputs.pVectorA);*/
+    /*Checks if the difference between velocity in previous periodic cycle and current periodic cycle. Considers it significant if the difference is higher than 4.5. Then checks if vectorDiff is greater than 0.7 than the previous velocity*/
```

Comment:
comment formatting is not great, can you reformat this and add a javadoc to the method